test/over_react/component/abstract_transition_test.dart

```diff
@@ -556,7 +556,7 @@ main() {
           // Have to use MutationObserver since setting the state to PRE_SHOWING
           // synchronously updates the state to SHOWING.
           var transitionAttrMutations = <List<String>>[];
-          var observer = new MutationObserver((records, observer) {
+          var observer = new MutationObserver((List<dynamic> records, observer) {
```

Comment:
Could use a comment that this is `<dynamic>` to work around a Dart HTML internals issue in the DDC